SU-HSDO/suhumsci
# SHS-5628: Discovery: Investigate best way to run VRT on all color pairings

# READY FOR REVIEW

## Summary
Update backstop version and configuration

## Need Review By (Date)
05/23

## Urgency
medium

## Steps to Test
1. In the theme folder, execute the following commands:
    - `npm run backstop:init`
    - `npm run backstop:reference`
    - `npm run backstop:test`
2. Confirm that the VRT runs as expected


## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
